### ISSUE TITLE
TTS reads out html code in messages

### ISSUE THREAD

**Issue by spanishpear** [kennib/necsus#66]
Ideally, html should be skipped and not read aloud

https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags might come in handy to use regex to check if tehre is html (this is a joke(

**Comment by cyphar** [comment]
Yeah we just TTS `message.text` when we should maybe TTS the rendered `innerText` version? (I also just checked and `innerText` does appear to strip everything.
